io.github.anirbandey1.ChatGPTElectron.yml

```diff
@@ -0,0 +1,60 @@
+app-id: io.github.anirbandey1.ChatGPTElectron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+rename-desktop-file: chatgpt-electron.desktop
+rename-icon: chatgpt-electron
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+modules:
+
+  - name: run
+    buildsystem: simple
+    build-commands:
+      - install -D run.sh /app/bin/run.sh
+    sources:
+      - type: script
+        dest-filename:  run.sh
+        commands:
+          - 'echo "Running Inside Sandbox" '
+          - 'export TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID}"'
+          - 'exec zypak-wrapper /app/opt/chatgpt-electron/chatgpt-electron'
+
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - cp -r usr/share /app
+      - desktop-file-edit --set-key=Exec --set-value="run.sh" /app/share/applications/chatgpt-electron.desktop
+      - desktop-file-edit --set-key=Icon --set-value="io.github.anirbandey1.ChatGPTElectron" /app/share/applications/chatgpt-electron.desktop
+
+    sources:
+      - type: git
+        url: "https://github.com/anirbandey1/chatgpt-electron-assets.git"
```

Comment:
so why isn't the appstream file there too?

Reply:
Appstream file is present along side the yaml file so  using it as file source

Reply:
as I said it would be better in that "asset" repository, whose content should be in the main one, but that last part doesn't matter here. It will just make your life easier.

Reply:
now, I have moved all the assets from the "assets" repo to the main [repo](https://github.com/anirbandey1/chatgpt-electron).